munininfluxdb/commands/fetch.py

```diff
@@ -1,31 +1,22 @@
 #!/usr/bin/env python
```

Comment:
This is one of the subcommands of the main command. It is loaded in `munininfluxdb/main.py`.